src/aap_eda/api/views/activation.py

```diff
@@ -115,11 +115,13 @@ def create(self, request):
         activation["rules_fired_count"] = 0
 
         if response.is_enabled:
-            activate_rulesets.delay(
+            job = activate_rulesets.delay(
                 is_restart=False,
                 activation_id=response.id,
                 deployment_type=settings.DEPLOYMENT_TYPE,
             )
+            response.current_job_id = job.id
```

Comment:
@hsong-rh Is this current_job_id the same as the RQ Job ID, if we have this ID could we retrieve the Job Data from the RQ Job?